lib/bn.js

```diff
@@ -450,6 +450,11 @@ BN.prototype.toString = function toString(base, padding) {
   }
 };
 
+BN.prototype.toNumber = function toNumber() {
+  assert(this.bitLength() <= 53, 'Number can only safely store up to 53 bits');
+  return parseInt(this.toString());
```

Comment:
Ah, I've assumed parseInt is radix 10 by default, but it is radix 8 if it starts with '0'?